examples/pylab_examples/pcolor_demo.py

```diff
@@ -20,7 +20,7 @@
 
 
 plt.subplot(2, 2, 1)
-plt.pcolor(x, y, z, cmap='RdBu', vmin=z_min, vmax=z_max)
+plt.pcolor(x, y, z, vmin=z_min, vmax=z_max)
```

Comment:
The data here does appear to be divergent, so I'm not sure switching to viridis is a good idea here.